src/MuTalk-Tests/MutationTestingAnalysisTest.class.st

```diff
@@ -166,13 +187,16 @@ MutationTestingAnalysisTest >> testExecutingTwoMutantsFromDiferentMethods [
 { #category : 'testing' }
 MutationTestingAnalysisTest >> testExecutingTwoMutantsFromSameMethod [
 	"Obtain 2 mutants using --->+ replacement in only one method."
+
 	| analysis results operatorSelection |
 	operatorSelection := OrderedCollection new.
 	operatorSelection addAll: self minusForPlusReplacementMutation.
-	analysis := MutationTestingAnalysis 
-		testCasesFrom: Array new
-		mutating: (Array with: AuxiliarClassForMutationTestingAnalysis)
-		using: operatorSelection.
+	analysis := MutationTestingAnalysisBuilder new
+		            testClasses: Array new;
+		            classesToMutate:
+			            (Array with: AuxiliarClassForMutationTestingAnalysis);
+		            operators: operatorSelection;
+		            buildAnalysis.
```

Comment:
This code looks repeated in many tests. Could we extract it?

Also, idiomatically I'd prefer `#()` for an empty array, but that's just a matter of taste :)